src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -74,22 +74,27 @@ function calculateAudioScore(
     return roundTripTime;
   };
 
+  const getDelay = (): number => {
+    const roundTripTime = getRoundTripTime();
+    const delay = (roundTripTime * MS_PER_SEC) / 2;
+    return delay || DEFAULT_DELAY;
+  };
+
   const audioScore = (packetLossRatio: number) => {
-    const LOCAL_DELAY = 20; // 20 msecs: typical frame duration
+    const LOCAL_DELAY = 30; // 30 msecs: typical frame duration
     const h = (x: number): number => x < 0 ? 0 : 1;
     const a = 0; // ILBC: a=10
     const b = 19.8;
     const c = 29.7;
-    const roundTripTime = getRoundTripTime();
-
+    const delay = getDelay();
     /**
      * Calculate the transmission rating factor, R
      */
     const calculateR = (): number => {
-      const d = roundTripTime + LOCAL_DELAY;
-      const delayImpairment = ((0.024 * d) + 0.11) * (d - 177.3) * h(d - 177.3);
-      const equipmentImpairment = (a + b) * Math.log(1 + (c * packetLossRatio));
-      return 94.2 - delayImpairment - equipmentImpairment;
+      const d = delay + LOCAL_DELAY;
+      const delayImpairment = 0.024 * d + 0.11 * (d - 177.3) * h(d - 177.3);
```

Comment:
Fixed the separation of terms regarding the original formula:

<img width="291" alt="Screen Shot 2020-09-07 at 11 32 50 AM" src="https://user-images.githubusercontent.com/10964589/92397858-e2c25480-f0fd-11ea-8aaa-1147c7b69b42.png">